README.md

```diff
@@ -72,6 +72,9 @@ Create a new machine key for your server:
 
 ### 2. Prep the server
 
+Some servers allow the root user to login. If your server is such a server, you
```

Comment:
I'm not sure this sentence adds any value. It's not necessary, because changing the root password is a reasonable recommendation regardless of whether you allow root logins. And it's confusing in this location, because it seems to apply to all the steps under "prep the server," but it doesn't -- you need to create an account for Ansible regardless, and authorizing your ssh key is unrelated to whether you allow root logins.